gen3release-sdk/gen3release/config/env.py

```diff
@@ -159,6 +157,8 @@ def _replace_all_versions(self, version, override, json_block):
             logging.debug("Malformed override json string passed - {}".format(override))
             dict_override = {}
         for svc in json_block:
+            if "dashboard" not in json_block:
```

Comment:
should we add this check for audit-service too? not sure if it is in all the commons yet

Reply:
I believe some of the commons do have audit-service deployed to them, but I am guessing all the envs related to that particular commons would use audit service. But for `dashboard`, some of the preprod envs dont use them. so when we do code promotion, `dashboard` is removed while making the PR to cdis-manifest.